astropy/nddata/nduncertainty.py

```diff
@@ -64,7 +75,7 @@ def parent_nddata(self):
     def parent_nddata(self, value):
         self._parent_nddata = value
 
-    @property
+    @abc.abstractproperty
```

Comment:
This is made an abstract property so that it can be implemented as a class attribute, right?